test/public/defaults.js

```diff
@@ -477,14 +480,14 @@ module.exports.checkColumnBalloon = async (page, rowIndex, columnIndex) => {
 /**
  * Check that a given cell of the given column displays the correct color depending on the status
  *
- * @param {{$: function}} page the puppeteer page
+ * @param {puppeteer.Page} page the puppeteer page
  * @param {number} rowIndex the index of the row to look for status color
  * @param {number} columnIndex the index of the column to look for status color
  * @returns {Promise<Chai.Assertion>} void promise
  */
 module.exports.checkEnvironmentStatusColor = async (page, rowIndex, columnIndex) => {
-    const cellStatus = await page.$(`tbody tr:nth-of-type(${rowIndex}) td:nth-of-type(${columnIndex})`);
-    const cell = await page.$(`tbody tr:nth-of-type(${rowIndex})`);
+    const cellStatus = await page.waitForSelector(`tbody tr:nth-of-type(${rowIndex}) td:nth-of-type(${columnIndex})`);
+    const cell = await page.waitForSelector(`tbody tr:nth-of-type(${rowIndex})`);
     const cellStatusContent = await getInnerHtml(cellStatus);
```

Comment:
I think it's `row`, but not a `cell`

Reply:
Whole function was flawed but not in the scope of this PR. Refactored anyway